mediaplatform_jwp/jwplatformapi/deliveryapi.py

```diff
@@ -16,7 +16,7 @@
 import jwt
```

Comment:
suggest naming the parent directory "api" as it is hard to imagine use talking a non JW api in the mediaplatform_jwp app.

Reply:
I agree